docs/reference/inference/put-inference.asciidoc

```diff
@@ -266,8 +303,18 @@ maximum token length. Defaults to `END`. Valid values are:
 
 `user`:::
 (optional, string)
-For `openai` service only. Specifies the user issuing the request, which can be
-used for abuse detection.
+For `openai` and `azureopenai` service only. Specifies the user issuing the
+request, which can be used for abuse detection.
+
+=====
++
+.`task_settings` for the `completion` task type
+[%collapsible%closed]
+=====
+`user`:::
+(optional, string)
+For `openai` service only. Specifies the user issuing the request, which can be used for abuse detection.
+>>>>>>> c5d724dcc1c (Update docs for Azure OpenAI Embeddings inference)
```

Comment:
```suggestion
```

Reply:
🤦 Thanks for the catch again... I need to check my conflict editing settings 😅